charts/newrelic-logging/values.yaml

```diff
@@ -134,3 +134,8 @@ nrStaging: false
 daemonSet:
   # Annotations to add to the DaemonSet.
   annotations: {}
+
+# When low data mode is enabled only minimal attributes are added to the logs. Kubernetes labels and
+# annotations are not included.
```

Comment:
Please add a small comment indicating that this also merges `plugin.type`, `plugin.version` and `plugin.source` together into a minified `plugin.source` field.